src/Parser/HavocDemonHunter/Modules/Statistics/Spells/Momentum.js

```diff
@@ -14,23 +15,26 @@ class Momentum extends Module {
     combatants: Combatants,
   };
 
+  on_initialized() {
+    if (!this.owner.error) {
+      this.active = this.combatants.selected.hasTalent(SPELLS.MOMENTUM_TALENT.id);
+    }
+  }
+
   statistic() {
 
     const momentumUptime = this.combatants.selected.getBuffUptime(SPELLS.MOMENTUM_TALENT.id);
 
     const momentumUptimePercentage = momentumUptime / this.owner.fightDuration;
 
-    if(momentumUptime > 0)
-    {
     return (
       <StatisticBox
         icon={<Icon icon="ability_foundryraid_demolition" alt="Momentum" />}
```

Comment:
Please use `SpellIcon` here to users can consistently hover the icon for tooltips about the spells to learn more about them.